contributing/index.md

```diff
@@ -22,3 +22,48 @@ If you want to get in touch with us first before contributing, you can use:
 
 * [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
 * [Strimzi Dev mailing list](https://lists.cncf.io/g/cncf-strimzi-dev/topics)
+
+# How to become a maintainer
+
+The governance of the project is defined in the [GOVERNANCE.md](https://github.com/strimzi/strimzi-kafka-operator/blob/master/GOVERNANCE.md) file in the Strimzi Github repository, but in summary certain members of the community are the "maintainers" who decide the direction of the project. New maintainers can be elected by a ⅔ majority vote of the existing maintainers.
+
+ So as to be transparant and to ensure that all potential maintainers are judged fairly and consistently the following criteria should be taken into account in electing new maintainers:
+
+## Mandatory criteria
+Evidence of _all_ of the following are required:
+
+* Sustained contributions over at least 3 months including at least two non-trivial PRs.
+* An area of developing expertise. The candidate does not have to be the _go-to_ expert, but they should have a more than superficial understanding of at least one area. For example:
+    - A particular operator
+    - Some Kubernetes concern (CRDs, API, JSON Schema etc).
+    - Some associated bit of Kafka technology in which the project has a strategic interest (e.g. Cruise Control)
+    - System tests
+
+## Additional criteria
```

Comment:
I would probably remove the title and the followup sentence and merge the two lists into one.